tests/sentry/api/endpoints/test_project_templates_index.py

```diff
@@ -55,3 +58,70 @@ def test_get__no_access(self):
         # Ensure this errors with 403, as the user does not have access to the organization
         response = self.get_error_response(org_two.id, status_code=403)
         assert response.status_code == 403
+
+
+class ProjectTemplateIndexPostTest(APITestCase):
+    endpoint = "sentry-api-0-organization-project-templates"
+
+    def setUp(self):
+        super().setUp()
+        self.user = self.create_user()
+        self.organization = self.create_organization(owner=self.user)
+        self.team = self.create_team(members=[self.user])
+
+        self.login_as(self.user)
+
+        self.project_template = self.create_project_template(organization=self.organization)
+        self.project_template_two = self.create_project_template(organization=self.organization)
+
+    def test_post__no_feature(self):
+        response = self.get_error_response(self.organization.id, method="POST", status_code=404)
+        assert response.status_code == 404
+
+    @with_feature(PROJECT_TEMPLATE_FEATURE_FLAG)
+    def test_post__as_member(self):
```

Comment:
I think this could benefit from a docstring - is this a 403 because the user isn't an orgmember?

Reply:
haha i had this as `non_admin` before but seiji asked me to change it to this.. i wanted to just convey that they don't have the correct permissions.

as far as the doc string.. idk, i feel like it's not sure helpful (am i the only dev that doesn't read inline docs? 😅). but since both you and seiji called this one out explicitly i feel like it's not clear and we should do something...

i wonder if it's something like `test_post__no_access` or something simple like that?

Reply:
lol I read them 😢 maybe `test_post__permissions`?

Reply:
hahah okokok, i can add a quick note to outline the setup and how it's working.

for the name, i feel like permissions is kinda a piece of what i'm testing, but not the whole aspect to it.. it's like `test_post__as_member_without_permission`?